src/components/cc-pricing-product/cc-pricing-product.js

```diff
@@ -99,12 +98,7 @@ export class CcPricingProduct extends withResizeObserver(LitElement) {
     */
     this.temporalities = DEFAULT_TEMPORALITY_LIST;
 
-    /** @type {number|null} Set by the `withResizeObserver` mixin. The width of the component in `px`. See the `onResize` method for more info. */
-    this._size = null;
-  }
-
-  onResize ({ width }) {
-    this._size = width;
+    new ResizeController(this);
```

Comment:
issue: let's keep a reference to the controller so we can get its `width` property

```js
this._resizeController = new ResizeController(this);
```